core/src/test/java/org/frankframework/util/MiscTest.java

```diff
@@ -44,32 +44,41 @@ public void testToFileSizeForValueDefaultValue() {
 	@Test
 	public void testNegativeToFileSize() {
 		String size = Misc.toFileSize(-1);
-		assertEquals("-1", size);
+		assertEquals("-1B", size);
```

Comment:
Volgens mij klopt dit niet? Ik zou verwachten dat -1 ook -1 blijft, of misschien wel een lege string.

Reply:
Tjah, wat als je een methode hebt die wil weten hoeveel bytes nodig zijn (en misschien tekort komen), dan zou dat een negatief getal kunnen zijn. 
Ik weet niet of dit iets is wat we expres af willen dwingen of net zo goed er in kunnen laten als het werkt. Dan is het ook de vraag of we alles niet gewoon absoluut maken.

Reply:
Als een resource niet weet hoe groot iets is, geeft hij -1 terug. De usecase hoeveel bytes nodig zijn zie ik niet?

Reply:
Hmm alrighty, maar zou de human readable variant daarvan dan niet juist "unknown" moeten zijn?

Reply:
```java
                String size = Misc.toFileSize(-1);
		assertEquals("unknown", size);
```

Reply:
Ja, eigenlijk zou ik dan null verwachten oid, maar ik weet niet wat er dan allemaal omvalt!